packages/init/__tests__/starters/documents.test.ts

```diff
@@ -55,20 +55,20 @@ describe("readDocuments", () => {
       title_POSITION: 1,
       slugs_INTERNAL: ["hello"],
       uids_INTERNAL: [],
-    });
+    };
 
     mockfs({
```

Comment:
what does mockfs stand for?

Reply:
it's a library (mock-fs) used to mock the file system (whenever fs is used). Really nice didn't know it either